README.md

```diff
@@ -502,6 +503,38 @@ Supported error codes:
 * `pb_utils.TritonError.UNAVAILABLE`
 * `pb_utils.TritonError.UNSUPPORTED`
 * `pb_utils.TritonError.ALREADY_EXISTS`
+* `pb_utils.TritonError.CANCELLED` (since 23.10)
+
+#### Request Cancellation
+
+One or more requests may be cancelled during execution, for example, cancelled
+by the user. Starting from 23.10, `request.is_cancelled()` returns up-to-date
+`True` or `False` on whether the request is cancelled. If a request is
+cancelled, the model should respond `pb_utils.TritonError.CANCELLED` in place of
+the normal output tensors on the request. For example:
+
+```python
+import triton_python_backend_utils as pb_utils
+
+class TritonPythonModel:
+    ...
+
+    def execute(self, requests):
+        responses = []
+
+        for request in requests:
+            if request.is_cancelled():
```

Comment:
I think a decoupled example will be a better choice of demonstrating the power of checking request cancellation. You should use `response_sender.is_cancelled()` in the same. 
May be add an example in the Decoupled mode section below.

Reply:
I added
```
Starting from 23.10, request cancellation can be checked directly on the
`InferenceResponseSender` object, for example `response_sender.is_cancelled()`,
even after the request has gone out-of-scope. If
`response_sender.is_cancelled()` returned `True`, the
TRITONSERVER_RESPONSE_COMPLETE_FINAL flag is sent automatically.
```
under decoupled section. Do you think it is enough?